### ISSUE TITLE
Failed to load the "vagrant-managed-servers" plugin on a freshly installed ubuntu server

### ISSUE THREAD

**Issue by davidhadas** [tknerr/vagrant-managed-servers#56]
On a fresh install of 14.04 server version 64bit
uname -a
Linux  3.19.0-25-generic 26~14.04.1-Ubuntu SMP Fri Jul 24 21:16:20 UTC 2015 x86_64 x86_64 x86_64 GNU/Linux

after 
creating a passwordless ssh and sudo for a user named "vcap" 
sudo apt-get install git vagrant ruby-full bundler  zlib1g-dev virtualbox
vagrant plugin install vagrant-managed-servers

--> cat Vagrantfile
Vagrant.configure("2") do |config|
  config.vm.box = "tknerr/managed-server-dummy"

  config.vm.provider :managed do |managed, override|
    managed.server = <my host IP>
    override.ssh.username = "vcap"
    override.ssh.private_key_path = "/home/vcap/.ssh/id_rsa"
  end
end

I get: 
--> vagrant up --provider=managed
Failed to load the "vagrant-managed-servers" plugin. View logs for more details.

The provider 'managed' could not be found, but was requested to
back the machine 'default'. Please use a provider that exists.

**Comment by davidhadas** [comment]
--> vagrant up --provider=managed --debug
ERROR root: Failed to load plugin: vagrant-managed-servers
ERROR root:  -- Error: #<RuntimeError: The Vagrant ManagedServers plugin is only compatible with Vagrant 1.6+>
ERROR root:  -- Backtrace:
ERROR root: /home/vcap/.vagrant.d/gems/gems/vagrant-managed-servers-0.7.1/lib/vagrant-managed-servers/plugin.rb:10:in `<top (required)>'

 --> vagrant -v
Vagrant 1.4.3

Bottom line - this is due to vagrant version being too old.
